docs/design/architectural-overview-v1.md

```diff
@@ -84,6 +84,31 @@ The rate limit counters can also be shared and used by different clusters in ord
 
 Shown above is a multi-cluster multi ingress gateway topology. This might be used to support a geographically distributed system for example. However, it is also possible to leverage overlay networking tools such as [Skupper](https://skupper.io) that integrate at the Kubernetes service level to have a single gateway cluster that then integrates with multiple backends (on different clusters or in custom infrastructure).
 
+### Observability
+
+The Kuadrant architecture is intended to work with some popular monitoring tools for tracing, metrics and log aggregation.
+Those tools are:
+
+- [Prometheus](https://prometheus.io/) for scraping metrics - and optionally [Thanos](https://github.com/thanos-io/thanos) for high availability & federation
```

Comment:
Do we want to mention Alerts with Prometheus not just metrics? We have some example alerts with more to come

Reply:
```suggestion
- [Prometheus](https://prometheus.io/) for scraping metrics & alert rule evaluation - and optionally [Thanos](https://github.com/thanos-io/thanos) for high availability & federation
```

@R-Lawton good catch. This sufficient for here?

Reply:
Yeah i think thats perfecto, Im thinking maybe added to the diagrams too, but ill leave that up to what you think